code/style.css

```diff
@@ -1,7 +1,339 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+/*The heading session*/
+.container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.bg-video {
+  position: absolute;
+  object-fit: cover;
+  max-width: 100%;
+  min-width: 100%;
+  z-index: 0;
+  min-height: 100%;
+  max-height: 100vh;
+}
+
+.button{ 
+  border: white;
+  background-color: #a8a377;
+  color: white;
+  font-weight: bold;
+  position: absolute;
+  border-radius: 5px;
+  z-index: 2;
+  top: 20px;
+  right: 10px;
+  font-size: 20px;
+}
+
+.button:hover {
+  border: solid #a8a377 1px;
+  color: #a8a377;
+  background-color: transparent;
+}
+
+.slogan {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  z-index: 1;
+  font-family: 'Syncopate', sans-serif;
+  width: 60px;
+  color: white;
+}
+
+.logo {
+  position: absolute;
+  z-index: 2;
+  margin-top: 35vh;
+  width: 170px;
+  height: auto;
+}
+
+
+/* Form and FAQ section*/
+.form-and-faq-container {
+  display: flex;
+  flex-direction: column;
+  background-color: teal;
+  margin-top: 0;
+}
+
+.form-header {
+  font-family: 'Syncopate', sans-serif;
+  font-size: 25px;
+}
+
+/*The Form*/
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: lightblue;
+  border-radius: 5px;
+  height: 900px;
+  margin: 10px;
+  
+}
+.contact-details {
+  margin-bottom: 30px;
+}
+
+label { 
+  display: block;
+  margin: 15px
+}
+
+input {
+  display: block;
+  font-size: 20px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 2px 5px 0 rgba(56, 43, 82,0.5)
+}
+
+.guest-input {
+  display: block;
+  border: none;
+  border-radius: 5px;
+}
+
+select {
+  font-size: 20px;
+  margin: 5px;
+}
+
+input:focus {
```

Comment:
Great that you've succeeded with Red level requirement and added focus on input tag